### ISSUE TITLE
Ability to set log level

### ISSUE THREAD

**Issue by Tethik** [Netflix-Skunkworks/raven-python-lambda#21]
I haven't found a way to be able to set separate log levels for the sentry logging. In my project I would prefer to be able to log only `errors` and `warnings` to sentry, while still logging `info` messages to stdout.

Relevant code:
https://github.com/Netflix-Skunkworks/raven-python-lambda/blob/c2eef75ce4b0e930b0828e1c4117026f72b02164/raven_python_lambda/__init__.py#L113

 I guess I could configure this separately without using this plugin, but I think it would be better if there was an optional environment variable which could set the log level for the sentry log handler.

**Comment by jwkvam** [comment]
I tried to address this in #19 among other things. I added a new env `SENTRY_LOG_LEVEL`.

**Comment by Tethik** [comment]
Cool, I checked your code and that is basically just what I wanted. Hoping now for the merge.

**Comment by kevgliss** [comment]
Merged in #19

**Comment by trptcolin** [comment]
I've got the same use case here (separate log levels for Sentry vs. everything else), but have a couple question on using it:
1. As far as I can tell, the code requires the `SENTRY_LOG_LEVEL` to be an integer like `SENTRY_LOG_LEVEL=40`, not a string like `SENTRY_LOG_LEVEL=ERROR` or `SENTRY_LOG_LEVEL=logging.ERROR` - is that the intended environment variable API?
2. Even with the default value of `logging.WARNING`, my app is still sending INFO logs to Sentry. This one could be me misunderstanding how python logging handlers / levels work, but here's my setup that ends up sending `logging.getLogger().info("stuff")` messages to Sentry.

```
import logging
logging.basicConfig()
logger = logging.getLogger()
logger.setLevel(logging.INFO)

from raven_python_lambda import RavenLambdaWrapper

@RavenLambdaWrapper()
def hander(event, context):
   ...stuff...
```

Shouldn't that be using the default of WARNING? Or am I misunderstanding?

**Comment by jwkvam** [comment]
1. I wrote the code for that. That was my intention, I can't speak for others.
2. Are you talking about sentry capturing the logs as issues or breadcrumbs? The following links might be helpful:
- https://docs.sentry.io/clients/python/breadcrumbs/
- https://docs.sentry.io/clients/python/integrations/logging/

**Comment by Tethik** [comment]
It's been a while since I worked on our serverless code, but I will also check tomorrow how we configure our logging.

Re: 1. That does seem a bit odd to use integers instead of a string to configure the level.

**Comment by mikegrima** [comment]
https://stackoverflow.com/questions/10332748/python-logging-setlevel
^^ Yup, we should change this behavior. PR is welcome :)

**Comment by trptcolin** [comment]
OK, PR raised for item 1 (preserving backwards compatibility).

On item 2, thanks @jwkvam - I hadn't dug into this breadcrumbs feature. As far as I can tell those only show up in the Sentry interface under the "Breadcrumbs" label within an Issue, not at the top level Issue. Sound right? For now I'm assuming so and digging further into how python logging works.

**Comment by Tethik** [comment]
Just to check back in: I'm still using an older version of this library in my projects. 

Added a comment to your PR :+1:

**Comment by trptcolin** [comment]
Just to close the loop: item 2 was fixed by 16965dcdf25e39f5b8a67d6f8f3cb3982e6b6fde (we were using 0.1.5), and 0.1.7 fixes it.
